app/views/layouts/application.html.erb

```diff
@@ -101,6 +101,17 @@
   <div class="footer">
     <div class="container">
       <div class="footer_info">
+
+<ul class="footernav">
```

Comment:
It might make sense to pull this out into a _partial, since we're using it in the header and the footer!